source/extensions/filters/http/original_src/original_src.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "envoy/http/filter.h"
+#include "envoy/network/address.h"
+
+#include "common/common/logger.h"
+
+#include "extensions/filters/http/original_src/config.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace OriginalSrc {
+
+/**
+ * Implements the Original Src http filter. This filter places the source address of the requested,
```

Comment:
It's meant to be `request`, not `requested`. The implication here is that it is actually distinct from the connection's address, since we may infer it from a property of the stream itself (proxy protocol, xff, etc). That's handled for us by downstreamRemoteAddress.

How does rephrasing it to `the downstream source address of the request, as determined by...` sound?

Reply:
Yeah that sounds good to me